packages/events/defaultSubscribers/dashboard.js

```diff
@@ -0,0 +1,44 @@
+const ora = require("ora");
+
+module.exports = {
+  initialization: function () {
+    this.logger = (...args) => {
+      if (this.quiet) return;
```

Comment:
Is the omission of `{}` in this situation something that's existing style? I personally hate this sort of thing.

Reply:
Yes, existing style. I also dislike it, though I can live with it as long as it's all one line. The pain comes in when the branched statement is on a new line and someone tries adding a line, thinking it's in a block.

Reply:
I'll fix it, and replace the occurrences in the other default subscribers.